### PR TITLE
Apply patch

### PR DESCRIPTION
**Pull request by SharonStrats**
checking

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,17 +54,19 @@
   },
   "homepage": "https://github.com/solidos/solid-ui",
   "dependencies": {
+    "@inrupt/universal-fetch": "^1.0.1",
```
Comment (SharonStrats) [inline package.json:57]: will leave it in dependencies instead of dev because I believe we will need it in other places so that we can use node 18.

Comment (SharonStrats) [reply package.json:57]: Thought I would leave it until we make a decision.

Note: this is a node 16 problem with the clearImmediate function affecting the autocompleteField tests.

File: package.json [hunk 2]
```diff
@@ -105,7 +107,7 @@
     "html-webpack-plugin": "^5.5.0",
     "husky": "^7.0.4",
     "jest": "^27.5.1",
-    "jsdom": "^19.0.0",
+    "jsdom": "^21.1.1",
```
Comment (SharonStrats) [inline package.json:110]: This is something I tried, the tests still work with the upgrade. But we might want to revert back so wanted to call it out.

**Comment by bourgeoa** [comment]
closed with https://github.com/SolidOS/solid-ui/pull/543/commits/8924ae81610399e968faf5b83a2f5669db70c87c

**Pull request closed without merging**